project-news-site/code/style.css

```diff
@@ -0,0 +1,147 @@
+body{
+background-image: url("https://tinyurl.com/yckuujut");
+}
+header {
+ display: flex;
+  width: 100%;
+  height: 100px;
+  /*background-color: lightgrey;*/
+  justify-content:space-evenly;
+  
+}
+
+.logo {
+font-size: 50px;
+font-family: 'Anton', sans-serif;
+padding: 15px;
+display:flex;
+margin-left: 25px;
+margin-right: 25px;
+display: none;
+}
+
+.links {
+  display:grid;
+  padding: 20px;
+  font-family: 'Anton', sans-serif;
+  justify-content: space-evenly;
+  color: black;
+  
+}
+a {
+    color: black;
+  } 
+  
+  a:hover {
+  color: green;
+  
+  }
+.big-news {
+display: grid;  
+grid-template-columns: 50% 50%;
+border: 10px;
+align-items: center;
+width: 100%;
+}
+
+.big-news img {
+  width: 100%
+
+
+}
+
+h1 {
+  font-size: 40px;
+  font-family: 'Anton', sans-serif;  
+  }
+.big-news-text{
+  display: grid;
+  flex-direction: row;
+}
+
+.test1 {

```

Comment:
There's a way of making the css more compressed, it is by putting all the test1 - test4 on the same row when they all are being targeted the same. I would suggest using that method for a more clean code